### PR TITLE
Add storage_account_tls_version query for Ansible

### LINKED ISSUE #1636: Add storage_account_tls_version query for Ansible
### Platform
Ansible

### Provider
Azure

### Description
Ensure Azure Storage Account is using the latest version of TLS encryption

### PR DESCRIPTION
**Pull request by mcarvalhox**
This query evaluates if the latest version of TLS encryption (i.e. TLS1_2) is being used. 
Closes #1636

### CODE REVIEW

**Comment by markmishaevcx** [comment]
Scan submitted to Checkmarx

**Comment by markmishaevcx** [comment]
Logo<br />**Checkmarx SAST - Scan Summary & Details**
### Cx-SAST Summary
Total of 4 vulnerabilities<br />High&nbsp;**0 High**<br />Medium&nbsp;**3 Medium**<br />Low&nbsp;**1 Low**<br />Info&nbsp;**0 Info**<br />
### Violation Summary
Medium&nbsp;**3** **Medium**<br />
[View more details on Checkmarx UI](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049140&projectid=758)
### Cx-SAST Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[30](https://github.com/Checkmarx/kics/blob/feature/storage_account_tls_version/cmd/builder/main.go#L30) |Medium|Path_Traversal|cmd/builder/main.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049140&projectid=758&pathid=3)
[52](https://github.com/Checkmarx/kics/blob/feature/storage_account_tls_version/pkg/kics/service.go#L52) |Medium|Denial_Of_Service_Resource_Exhaustion|pkg/kics/service.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049140&projectid=758&pathid=1)
[83](https://github.com/Checkmarx/kics/blob/feature/storage_account_tls_version/test/main_test.go#L83) |Medium|Denial_Of_Service_Resource_Exhaustion|test/main_test.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049140&projectid=758&pathid=2)

File: assets/queries/ansible/azure/storage_account_tls_version/query.rego [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+} else = result {
+    result := [body | playbook := document.playbooks[_]; body := playbook ]
+    count(result) != 0
+}
+
+IsLatestVersion(version) {
+	version == "TLS1_2"
+} else {
+	version == true
+}
```
Comment (joaomartinscx) [inline assets/queries/ansible/azure/storage_account_tls_version/query.rego:49]: How does this else enforce that the minimum tls version to be used is the latest? Keep in mind that the default value is tls1_0

File: assets/queries/ansible/azure/storage_account_tls_version/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "storage_account_tls_version",
+  "queryName": "Storage Account TLS Version",
```
Comment (joaomartinscx) [inline assets/queries/ansible/azure/storage_account_tls_version/metadata.json:3]: The query name should identify the vulnerability or fault, in this case, something like: "Storage Account Is Not Using Latest TLS Encryption Version"

File: assets/queries/ansible/azure/storage_account_tls_version/metadata.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  "severity": "MEDIUM",
+  "category": "Storage",
```
Comment (joaomartinscx) [inline assets/queries/ansible/azure/storage_account_tls_version/metadata.json:5]: Refactor category name according to the list given

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

**Review by joaomartinscx — APPROVED** [review]
Everything looks good to me, good job!

**Pull request merged**